lib/validator/v3/validator_test.go

```diff
@@ -17,13 +17,13 @@ package v3_test
 import (
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this and next were added by `goimports`…

Reply:
Yeah, I've noticed it has started doing this recently. /shrug.